cellfinder_napari/train/train_containers.py

```diff
@@ -0,0 +1,124 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+
+from cellfinder_core.download.models import model_weight_urls
+from cellfinder_core.train.train_yml import models
+
+from cellfinder_napari.input_container import InputContainer
+from cellfinder_napari.utils import html_label_widget
+
+
+@dataclass
+class TrainingDataInputs(InputContainer):
+    """Container for Training Data input widgets"""
+
+    yaml_files: Path = Path.home()
+    output_directory: Path = Path.home()
+
+    def as_core_arguments(self) -> dict:
+        arguments = super().as_core_arguments()
+        arguments["output_dir"] = arguments.pop("output_directory")
+        arguments["yaml_file"] = arguments.pop("yaml_files")
+        return arguments
+
+    @classmethod
+    def widget_representation(cls) -> dict:
+        return dict(
+            data_options=html_label_widget("Training Data:"),
+            yaml_files=cls._custom_widget(
+                "yaml_files",
+                custom_label="YAML files",
+                mode="rm",
```

Comment:
Yes, these are options that are passed down to `napari`'s widget creation functionality, in this case a [`magicgui FileEdit`](https://napari.org/magicgui/usage/_autosummary/magicgui.widgets.FileEdit.html#magicgui.widgets.FileEdit) widget (It decides it's a FileEdit because we pass a `Path` as the value).

Looks like the ["rm" stands for "**r**eturn one or **m**ore existing files" ](https://napari.org/magicgui/api/types.html#magicgui.types.FileDialogMode) which we need here as we may want to specify a number of YAML files to be used.

Reply:
Good point. I now use the `magicgui.types` enum `FileDialogMode.EXISTING_FILES` directly which should make this clearer.